src/mozilla_taskgraph/actions/release_promotion.py

```diff
@@ -126,6 +127,12 @@ def release_promotion_action(parameters, graph_config, input, task_group_id, tas
     parameters["shipping_phase"] = input["release_promotion_flavor"]
     parameters["tasks_for"] = "action"
 
+    version_parser_objpath = "mozilla_taskgraph.version:default_parser"
+    if "version-parser" in graph_config:
+        version_parser_objpath = graph_config["version-parser"]
+    version_func = find_object(version_parser_objpath)
+    parameters["version"] = version_func(parameters)
```

Comment:
Should we sanitize this value in any way?

Reply:
I don't think we should do that here. We should check it when it's used as input to things like scriptworker tasks and the like.. but at this point the version could be pretty much any format.

Reply:
Would a list/object/non-string values be allowed?

Reply:
I was about to say that this will be validated against the parameters schema:
https://github.com/taskcluster/taskgraph/blob/main/src/taskgraph/parameters.py#L62

But looking at this, I don't think we ever call `parameters.check()` because `taskgraph_decision` only does that when it doesn't receive parameters already.

That said, I think this is a bug in `taskgraph` rather than `mozilla-taskgraph`.